packages/preview/sdu-typst-touying/0.1.0/example/main.typ

```diff
@@ -0,0 +1,402 @@
+#set text(font: "Source Han Serif", lang: "CN",region: "CN")
+#import "../lib.typ": *
```

Comment:
You should import the package using its full name and not a relative path, otherwise projects created from your template won't work.

```suggestion
#import "@preview/sdu-typst-touying:0.1.0": *
```

(this example is with the current name, you should replace it with the new one once you have made the change)